src/universal_transfer_operator/data_providers/filesystem/aws/s3.py

```diff
@@ -148,12 +144,12 @@ def s3_extra_args(self) -> Any:
         return self.dataset.extra.get("s3_extra_args", {})
 
     @property
-    def bucket_name(self) -> str:
+    def bucket_name(self) -> Any:
         bucket_name, _ = self.hook.parse_s3_url(self.dataset.path)
         return bucket_name
 
     @property
-    def s3_key(self) -> str:
+    def s3_key(self) -> Any:
```

Comment:
Why is this any?

Reply:
Answered here - https://github.com/astronomer/apache-airflow-provider-transfers/pull/55#discussion_r1161724308